sklearn/linear_model/coordinate_descent.py

```diff
@@ -279,21 +281,21 @@ def enet_path(X, y, l1_ratio=0.5, eps=1e-3, n_alphas=100, alphas=None,
     For mono-output tasks it is
 
     .. math::
-        \\frac{1}{2 * \\mathrm{n\\_samples}} * ||y - Xw||^2_2
-        + \\alpha * \\mathrm{l1\\_ratio} * ||w||_1
-        + 0.5 * \\alpha * (1 - \\mathrm{l1\\_ratio}) * ||w||^2_2
+        \\frac{1}{2 * \\mathrm{n\\_samples}} * \|y - Xw\|^2_2
+        + \\alpha * \\mathrm{l1\\_ratio} * \|w\|_1
+        + 0.5 * \\alpha * (1 - \\mathrm{l1\\_ratio}) * \|w\|^2_2
 
     For multi-output tasks it is
 
     .. math::
-        \\frac{1}{2 * \\mathrm{n\\_samples}} * ||Y - XW||^{Fro}_2
-        + \\alpha * \\mathrm{l1\\_ratio} * ||W||_{21}
-        + 0.5 * \\alpha * (1 - \\mathrm{l1\\_ratio}) * ||W||_{Fro}^2
+        \\frac{1}{2 * \\mathrm{n\\_samples}} * \|Y - XW\|^{\mathrm{Fro}}_2
```

Comment:
I'm not convinced that having 2 subscript and Fro superscript is correct here. Have you checked this against a reference?